joomla/joomla-cms
# [JInstaller::parseXMLInstallFile] Parse also metadata section of manifest files

### Summary of Changes

This PR allow JInstaller::parseXMLInstallFile to also parse the metadata section in xml files (ex: xml language files).
### Testing Instructions

Code review.
- add this to your isis/index.php and refresh the page you will see the array with that xml file.

``` php
print_r(JInstaller::parseXMLInstallFile(JPATH_SITE . '/language/en-GB/install.xml'));
```
- Now add this to the end of the en-GB `install.xml` file:

``` xml
    [...]
    </files>
    <metadata>
        <name>English (en-GB)</name>
        <tag>en-GB</tag>
        <rtl>0</rtl>
        <locale>en_GB.utf8, en_GB.UTF-8, en_GB, eng_GB, en, english, english-uk, uk, gbr, britain, england, great britain, uk, united kingdom, united-kingdom</locale>
        <firstDay>0</firstDay>
        <weekEnd>0,6</weekEnd>
    </metadata>
    <params />
</extension>
```
- Refresh page.  You will see the metadata section IS NOT there.
- Apply patch.
- Refresh page.  You will see the metadata section IS there.
- Install any extension, language to confirm all installs fine.
### Documentation Changes Required

None.
